docs/developer_docs/environment.rst

```diff
@@ -39,36 +47,20 @@ Now, every time you work on your development environment, just remember to switc
 .. _github: https://github.com/learningequality/ka-lite
 
 
-Running directly from source
-____________________________
-
-
-KA Lite can also be run as a "source distribution" for development purposes.
-By this, we just mean a git checkout (from `our github <https://github.com/learningequality/ka-lite/>`_).
-
-.. note:: Running directly from source will also maintain all user data in that
-          same directory! This is convenient for having several versions of
-          kalite with different data on the same computer.
-
-If you are able to use pip and install conventional python packages from an
-online source, then the quickest option to install the latest stable release
-of KA Lite is ``pip install ka-lite` or `pip install ka-lite-static``.
-
-
 Static vs. Dynamic version
 __________________________
 
 Apart from Python itself, KA Lite depends on a couple of python applications,
-mainly from the Django ecology. These applications can be installed in two ways:
+mainly from the Django ecology. These dependencies can be installed in two ways:
```

Comment:
Sure, sounds better... it wasn't a change introduced in this PR, if you like, you can change it by clicking "Edit on Github", top right: http://ka-lite.readthedocs.io/en/develop/developer_docs/environment.html